packages/aws-cdk-lib/aws-s3/lib/bucket.ts

```diff
@@ -1544,6 +1605,13 @@ export interface BucketProps {
    */
   readonly serverAccessLogsPrefix?: string;
 
+  /**
+   * Optional key format for log objects.
+   *
+   * @default - the default key format is: [DestinationPrefix][YYYY]-[MM]-[DD]-[hh]-[mm]-[ss]-[UniqueString]
+   */
+  readonly logObjectKeyFormat?: LogObjectKeyFormat;
+
```

Comment:
Would calling it `targetObjectKeyFormat` be better? Matches with the CFN property name and its essentially a pass through already.

Reply:
Thanks!
Come to think of this, you are right.
Renamed the property and the enum-like class name.

I was thinking about naming the `log` prefix to the `targetObjectKeyFormat` because the log settings are flat in the `BucketProps` interface and it would be confusing without the `log` prefix.